specification/relay/resource-manager/Microsoft.Relay/2017-04-01/relay.json

```diff
@@ -42,9 +42,9 @@
         ],
         "operationId": "Operations_List",
         "x-ms-examples": {
-          "RelayOperationsList": { "$ref": "./examples/RelayOperations_List.json" }
+          "RelayOperationsList": { "$ref": "../examples/RelayOperations_List.json" }
```

Comment:
@sethmanheim Why were these paths changed? They are broken now! @sarangan12 Note that both the `model` and `semantic` validator failed on this PR.

Reply:
@olydis That's pretty strange, I made sure to only edit "Description" fields. I guess I inadvertently must have changed these. Sorry. Can you fix, or you want me to?

Reply:
@olydis I believe this is because I originally pulled from master. I didn't realize we were merging to current. So I have filed PR #1675 to fix the errors.

Reply:
@sethmanheim Sorry for only getting back now, yes that makes sense, in master the structure was different... thanks a ton for the fix!